### PR TITLE
Fix for https://github.com/dwavesystems/dimod/commit/d1f12d4c294168afc0e5cd5768bb5a1b5fd13a3b

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
https://github.com/dwavesystems/dwave-ocean-sdk/pull/169#issuecomment-982758636

### CODE REVIEW

File: hybrid/samplers.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def next(self, state, **runopts):
             try:
                 num_tries += 1
                 response = self.sampler.sample(state.subproblem, **params)
-            except ValueError as exc:
+            except KeyError as exc:
```
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply hybrid/samplers.py:242]:
```if not adjacency.keys() >= bqm.variables:
            raise BinaryQuadraticModelStructureError(
                f"given bqm contains a variable, {(adjacency.keys() - bqm.variables).pop()!r}, "
```
The subtraction elements are swapped.

[the remark this answers is not in the document]
Comment (arcondello) [reply hybrid/samplers.py:242]: Agree with Rad that there is a bug in dimod here. I can fix that today.

[the remark this answers is not in the document]
Comment (randomir) [reply hybrid/samplers.py:242]: And given that `BinaryQuadraticModelStructureError` subclasses `ValueError`, fixing string format bug in dimod will fix the failing dwave-hybrid unittest.

**Comment by JoelPasvolsky** [comment]
Closed by https://github.com/dwavesystems/dimod/pull/1058

**Pull request closed without merging**